### ISSUE TITLE
Doesn't recognise rules with @typescript-eslint prefix

### ISSUE THREAD

**Issue by golergka** [denoland/deno_lint#926]
I've only just discovered this package and tried to use it as in-place replacement for ESLint. However, although the package readme says that it supports all `@typescript-eslint` rules, they seem to have different names here — without the prefix.

### Lint Name

`@typescript-eslint/no-explicit-any` or any other `@typescript-eslint` rule.

### Code Snippet

```ts
// eslint-disable-next-line @typescript-eslint/no-explicit-any
function isPoolClient(p: any): p is PoolClient {
```

### Expected Result

deno_lint recognizes `@typescript-eslint/no-explicit-any` rule as `no-explicit-any` rule.

### Actual Result

```
error[ban-unknown-rule-code]: Unknown rule for code "@typescript-eslint/no-explicit-any"
 --> /Users/golergka/Projects/pg-tx/src/client.ts:3:1
  |
3 | // eslint-disable-next-line @typescript-eslint/no-explicit-any
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
error[no-explicit-any]: `any` type is not allowed
 --> /Users/golergka/Projects/pg-tx/src/client.ts:4:26
  |
4 | function isPoolClient(p: any): p is PoolClient {
  |                          ^^^
  |
  = help: Use a specific type other than `any`
```

### Version

Used with node via @node-rs/deno-lint package, version 1.9.2.

**Comment by bartlomieju** [comment]
I thought we had this working, but looks like it doesn't. IIRC this somehow conflicted with other rules we have like `no-unused-ignore` and proposed solution was to add "aliases" to rule names (so a single rule can be known as `no-explicit-any` or `@typescript-eslint/no-explicit-any`). I guess this is still desirable to ease the migration from existing tools.

@magurotuna any thoughts on this?

**Comment by magurotuna** [comment]
@bartlomieju 
Adding aliases to rule names sounds great to me.

**Comment by golergka** [comment]
Thanks for clarifying that this is indeed intended behaviour! It actually wasn't completely sure if this linter is intended as a drop-in and highly compatible replacement of ESLint after reading the docs. If it is, would you accept a PR that clarified that? (Or may be I'm just bad at reading documentation and it's actually stated there loud and clear).
